Index: python-daemon-1.5.5/setup.py
===================================================================
--- python-daemon-1.5.5.orig/setup.py	2012-10-30 19:19:58.598192736 -0700
+++ python-daemon-1.5.5/setup.py	2012-10-30 19:37:12.809601462 -0700
@@ -15,7 +15,7 @@
     """
 
 import textwrap
-from setuptools import setup, find_packages
+from distutils.core import setup
 
 distribution_name = "python-daemon"
 main_module_name = 'daemon'
@@ -31,7 +31,7 @@
 setup(
     name=distribution_name,
     version=version.version,
-    packages=find_packages(exclude=["test"]),
+    packages=["daemon","daemon/version"],
 
     # setuptools metadata
     zip_safe=False,
